File: Plugins/Aspose_Pdf_Java_for_PHP/src/Aspose/Pdf/WorkingWithDocumentObject/GetPdfFileInfo.php
<?php

namespace Aspose\Pdf\WorkingWithDocumentObject;

use com\aspose\pdf\Document as Document;

class GetPdfFileInfo {

    public static function run($dataDir=null)
    {

        # Open a pdf document.
        $doc = new Document($dataDir . "input1.pdf");

        # Get document information
        $doc_info = $doc->getInfo();

        # Show document information
        print "Author:-" . $doc_info->getAuthor();
        print "Creation Date:-" . $doc_info->getCreationDate();
        print "Keywords:-" . $doc_info->getKeywords();
        print "Modify Date:-" . $doc_info->getModDate();
        print "Subject:-" . $doc_info->getSubject();
        print "Title:-" . $doc_info->getTitle();

    }
}

File: Plugins/Aspose-Pdf-Java-for-Jython/asposepdf/WorkingWithDocumentObject/GetPdfFileInfo.py
from asposepdf import Settings
from com.aspose.pdf import Document

class GetPdfFileInfo:

    def __init__(self):
        dataDir = Settings.dataDir + 'WorkingWithDocumentObject/GetPdfFileInfo/'
        
        # Open a pdf document.
        doc = Document(dataDir + "input1.pdf")

        # Get document information
        doc_info = doc.getInfo()

        # Show document information
        print "Author:"
        print doc_info.getAuthor()
        print "Creation Date:"
        print doc_info.getCreationDate()
        print "Keywords:"
        print doc_info.getKeywords()
        print "Modify Date:"
        print doc_info.getModDate()
        print "Subject:"
        print doc_info.getSubject()
        print "Title:"
        print doc_info.getTitle()
 
if __name__ == '__main__':        
    GetPdfFileInfo()